### DISCUSSION TITLE
Allow grouping by app in a monorepo

### DISCUSSION THREAD

**Discussion by jacobrask** [renovatebot/renovate#13901] [Request Help]
### What would you like Renovate to be able to do?

In a JS monorepo with a structure like this:

```
apps/
  first-app/
  second-app/
packages/
  shared-package-1
  shared-package-2
```

The different apps are maintained by different teams, and with separate `CODEOWNERS` per app. I would like Renovate to create one PR for every app, bumping only the dependencies in that app, instead of globally in the repository.

### If you have any ideas on how this should be implemented, please tell us here.

Could it be done with `packageRules` somehow?

### Is this a feature you are interested in implementing yourself?

No

**Comment by rarkins** [comment]
This is already possible using `additionalBranchPrefix`. See https://github.com/renovatebot/renovate/discussions/7993#discussioncomment-203894 for an example
